server/notification-providers/sevenio.js

```diff
@@ -0,0 +1,79 @@
+const NotificationProvider = require("./notification-provider");
+const Axios = require("axios");
```

Comment:
let^'s not be special here ^^

```suggestion
const axios = require("axios");
```